plugins/elantp/fu-elantp-hid-device.c

```diff
@@ -433,6 +433,11 @@ fu_elantp_hid_device_setup(FuDevice *device, GError **error)
 		g_autoptr(FuElantpHidHapticDevice) cfg = fu_elantp_haptic_device_new(device);
 		fu_device_add_child(FU_DEVICE(device), FU_DEVICE(cfg));
 	}
+
+	/* fix an unsuitable i²c name, e.g. `VEN 04F3:00 04F3:311C` */
+	if (g_str_has_prefix(fu_device_get_name(device), "VEN "))
+		fu_device_set_name(device, "Touchpad");
```

Comment:
How do you know it's not a touchscreen?  I think it's better to just call it "Human Interface Device" if you don't know.

Reply:
@jinglewu can you help here? Thanks!

Reply:
Elan touchpad VID 04f3 PID 3XXX.

Reply:
Thanks, patch updated.